package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "pretest": "npm run db:undo:migrate:test && npm run db:migrate:test && npm run db:seed:test",
-    "test": "cross-env NODE_ENV=test nyc mocha ./test --require @babel/register --exit --timeout 250000",
+    "test": "cross-env NODE_ENV=test nyc mocha ./test/index --require @babel/register --exit --timeout 250000",
```

Comment:
I think the index you added in the test script  ./test/index does not matter. mocha runs all the test file in the test folder. I stand to be corrected. thanks

Reply:
@el-Joft No, it doesn't run it automatically.

Reply:
Well, using the index file we can actually control the tests in the order in which we want all of it to occur. I imported all the tests in a way that it should run sequentially. For example, a user must be created before he can create an article. And an article must be created before it can be commented on. I feel like if we connect these units together and refactor our tests a little bit to depend on each other, we can then really have an integration test.